### DISCUSSION TITLE
Adding objects to the scene

### DISCUSSION THREAD

**Discussion by weirdooo-18** [NVlabs/sionna#368] [Q&A] [answered]
Hey all, 
           I was wondering if we can add objects to the scene, like cars, trees etc. , to make it more like a real-life scenario. I did browse through the documentation, but coudn't find a proper source to do this. 

Thank you.

**Comment by jhoydis** [comment, marked as answer]
Hi @weirdooo-18,

You can only add objects to a scene via Blender; not after it has been loaded in Mitsuba. Concerning trees, there is currently no model for foliage loss in Sionna. Concerning objects like cars, furniture, etc, you should avoid too detailed meshes.

**Reply by weirdooo-18** [reply]
Thank you sir.
